evm2wasm.cpp

```diff
@@ -1,11 +1,12 @@
 #include "wasm-binary.h"
 #include "wasm-s-parser.h"
+#include "evm2wast.h"
 
 using namespace std;
 
 namespace {
 
-string wast2wasm(string input, bool debug = false) {
+string wast2wasm(string input, bool debug = true) {
```

Comment:
If you really want to have debugging on by default please do that by the invocation.